euphoria-avro/build.gradle

```diff
@@ -0,0 +1,13 @@
+dependencies {
+  compile project(':euphoria-core')
+  compile project(':euphoria-hadoop')
+  compile project(':euphoria-spark')
```

Comment:
why do we need spark dependency here?

Reply:
removed and added the right dependencies, originaly spark used to test